File: Logistics.Data/ApprovalTemplate.cs
using System.Collections.Generic;

namespace Logistics.Data
{
    public class ApprovalTemplate
    {
        public int Id { get; set; }
        public int SiteId { get; set; }
        public string Name { get; set; }

        public Site Site { get; set; }

        public List<ApprovalTemplateGroup> ApprovalTemplateGroups { get; set; }
    }
}

File: Logistics.Web/ClientApp/src/app/models/approval-template.ts
import { ApprovalTemplateGroup } from './approval-template-group';
import { Site } from './site';

export class ApprovalTemplate {
  id: number;
  siteId: number;
  name: string;
  site: Site;
  approvalTemplateGroups: ApprovalTemplateGroup[];
}
